<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>796</width>
     <height>555</height>
    </rect>
   </property>
   <property name="windowTitle">
    <string>Oasis</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QMainWindow {
   background-color: #FFFFFF;
   border: 1px solid gray;
   padding: 5px;
 }</string>
   </property>
   <widget class="QWidget" name="centralwidget">
    <widget class="QLabel" name="cesOutput">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>380</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="sessionStart">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>430</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="audioInput">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="intensity">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>310</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="upIntensity">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>360</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="downIntensity">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>410</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="power">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>480</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="min20">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>81</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="min45">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>90</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="udSession">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>80</y>
       <width>121</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="MET">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>160</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="delta">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>160</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="alpha">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>160</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="hundredHZ">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>160</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>120</y>
       <width>271</width>
       <height>381</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QMenuBar" name="menubar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>796</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QStatusBar" name="statusbar"/>
  </widget>
  <resources/>
  <connections/>
 </ui>
